worker/src/dns.rs

```diff
@@ -0,0 +1,83 @@
+use crate::SPRUCE_DIDWEB;
+use anyhow::{anyhow, Result};
+use chrono::{SecondsFormat, Utc};
+use serde::Deserialize;
+use serde_json::json;
+use ssi::{blakesig::hash_public_key, jwk::JWK, vc::Credential};
+use url::Url;
+use uuid::Uuid;
+
+#[derive(Deserialize, Debug)]
+pub struct DnsResponse {
+    pub Answer: Vec<AnswerResponse>,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct AnswerResponse {
+    pub name: String,
+    pub data: String,
+}
+
+pub fn extract_dns_signature(tweet: String) -> Result<(String, String)> {
+    let mut sig_target = "".to_string();
+    for line in tweet.split('=').collect::<Vec<&str>>() {
```

Comment:
Be careful how you name your variables

Anyway I'm not a fan of the current format of the record. Following the example of Instagram I would be in favour of `tzprofiles-verification=edsig...` with the message being passed to the worker along with the request. In the future if we really wanted to embed the message it could go in its own record or find a way to add it nicely to the existing record.

Reply:
Got it. I followed this pattern and is now passing the message as a separate parameter in the request to worker.